### DISCUSSION TITLE
Editing Large CSS Files and Accessing Projects

### DISCUSSION THREAD

**Discussion by neutronltd** [brackets-cont/brackets#315] [Feature Request]
Issues arise when working with large CSS files in Brackets, primarily centered around the Search and Replace functionality and project loading times. When attempting to use the Search and Replace function within a sizable CSS file, it only locates one match at a time, significantly hindering efficiency. Additionally, as the file size increases, editing performance noticeably deteriorates, causing delays and sluggish responsiveness.

Furthermore, when accessing projects containing thousands of files, Brackets experiences significant slowdowns and prolonged loading times, making it cumbersome to navigate and work within such projects efficiently.

These performance limitations significantly impede productivity, especially for developers working on projects with extensive CSS files or large codebases. Addressing these issues would greatly enhance the usability and performance of Brackets, ensuring a smoother experience for developers dealing with substantial codebases and projects.

**Comment by abose** [comment]
Can you attach a sample large css file that is hard to edit to investigate. Large project support should be fixed in march release of phcode.

**Reply by neutronltd** [reply]
[lite-blue.zip](https://github.com/brackets-cont/brackets/files/14507208/lite-blue.zip)

Please check this file lite-blue.zip, inside has the CSS file with 32,000 lines of CSS code. 
The search and replace feature do not work on this file.

**Comment by charlypa** [comment]
@neutronltd,

Thanks for your feedback. We released an update for Phoenix Code yesterday aimed at fixing the issues you mentioned. Please download the latest version from https://phcode.io/ and test it out. Let us know your thoughts.

**Reply by neutronltd** [reply]
Thank you for your feedback. I'll check it.

**Reply by abose** [reply]
Just to clarify that large project support has been added, but the single large css file issue is yet to be fixed.
